pkg/maps/multicast/subscribermap.go

```diff
@@ -48,6 +48,9 @@ type GroupV4Map interface {
 // addresses.
 type GroupV4OuterMap struct {
 	*ebpf.Map
+
+	// BatchLookupSupported indicates if the kernel supports batch lookup.
+	BatchLookupSupported bool
```

Comment:
To be fair I'm not sure if it matters much but does this need to be an exported field? I would like to avoid other packages starting to depend on this, so that the replacement with a new API in cilium/ebpf is just a matter of internal replacement.

Reply:
good catch, this can be unexported type - I'll update the code.